### PR TITLE
adding command to remove docker container

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: scripts/launch_validator_node.sh [hunk 1]
```diff
@@ -91,6 +91,9 @@ function launch_nodes {
   
   # add-validator node
   become_a_validator
+
```
Comment (AndreiNavoichyk) [inline scripts/launch_validator_node.sh:94]: @karthikeyan-cere 
Double-check the ticket description. Container removing can be added to the run command.

Comment (karthikeyan-cere) [reply scripts/launch_validator_node.sh:94]: Updated command

Comment (AndreiNavoichyk) [reply scripts/launch_validator_node.sh:94]: Remove extra line here

Comment (AndreiNavoichyk) [reply scripts/launch_validator_node.sh:94]: @karthikeyan-cere 
Remove extra line.

**Review by AndreiNavoichyk — APPROVED** [review]

File: scripts/launch_validator_node.sh (hunk 1 above)
Comment (karthikeyan-cere) [reply scripts/launch_validator_node.sh]: Done.

**Pull request merged**